src/converse-muc-views.js

```diff
@@ -542,11 +547,17 @@ converse.plugins.add('converse-muc-views', {
                     '__': _converse.__,
                     '_converse': _converse,
                     'label_room_address': _converse.muc_domain ? __('Groupchat name') :  __('Groupchat address'),
-                    'chatroom_placeholder': placeholder
+                    'chatroom_placeholder': placeholder,
+                    'muc_roomid_policy_error_msg': this.muc_roomid_policy_error_msg,
+                    'muc_roomid_policy_hint': _converse.muc_roomid_policy_hint
                 }));
             },
 
             afterRender () {
+                if (_converse.muc_roomid_policy_hint) {
+                    this.el.querySelector('.roomid-policy-hint').innerHTML
+                        = _converse.muc_roomid_policy_hint;
+                }
```

Comment:
It's a bad idea to allow arbitrary HTML to be passed in to `converse.initialize` and then to render that since it creates a potential opening for malicious code to be injected.

I think the hint should just be text or alternatively you should use the `xss` library to filter out harmful markup and scripts (for example see converse-message-view.js).

Also, you should rather render the hint declaratively in the template, than using `innerHTML`.
If you use only two curly brackets, then the markup is rendered, three curly brackets cause text-only rendering.

Reply:
Again thanks for the review. Pushed the necessary changes.